src/components/Table.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+
+const ApiRender = ({post, ReactPaginate, pageNo, handlePageClick}) => {
```

Comment:
Improve naming convention for the whole PR. Name should depict what a variable contain, for the method it should describe what it do.

Function component name should be same as file name but in Upper camel Case.